joss_paper/paper.md

```diff
@@ -0,0 +1,456 @@
+
+The four most significant subpackages of `sunpy` are described below.
+
+The `sunpy.net` subpackage provides a unified interface that simplifies and homogenizes search and retrieval by querying and downloading data from many solar data sources, irrespective of the underlying data-source client. It currently supports sourcing data from 18 different space- and ground-based solar observatories.
+
+The `sunpy.map` and `sunpy.timeseries` subpackages provide core data types (`Map` and `TimeSeries`, respectively) that are designed to provide a general, standard, and consistent interface for loading and representing solar data across different instruments and missions. These classes load data which conform to solar physics standards and conventions such as FITS [@wells_fits_1981], FITS World Coordinate Systems (WCS) [@fits_wcs], and solar-specific FITS headers [@thompson_coordinate_2006], while allowing customization to account for differences in specific instruments. Visualization methods are also provided to inspect and plot those data. Example visulizations of both `TimeSeries` and `Map` are shown in Figure 1.
+
+![Left: An example of `TimeSeries` for the GOES X-ray Sensor in two broadband channels. Right: A `Map` of the extreme ultraviolet 171 $\mathring A$ channel of AIA corresponding to the time of a solar flare depicted by the vertical dashed line in the left-hand panel.](https://codimd.s3.shivering-isles.com/demo/uploads/upload_5eabd346e54c29b1d9b74aa11e351b30.png)
+
+The `sunpy.coordinates` subpackage provides support for representing and transforming coordinates used in solar physics and astrophysics. These coordinates may represent events (e.g., flares), features on or above the Sun (e.g., magnetic loops), or the position of structures traveling throughout the heliosphere (e.g., coronal mass ejections). The package currently implements the most widely used Sun-centered coordinate frames, and extends `astropy.coordinates`.
+
+Other functionality provided by `sunpy` includes physical models of solar behavior, such as differential rotation, color maps for certain data sources, image-processing routines integrated with `Map`, and useful physical parameters such as constants.
+
+The `sunpy` package is designed to be extensible, which means that it is easy to add support for additional instruments or data sources. It relies heavily on the `astropy`[@astropy] Python package as well as the scientific python stack (e.g. `numpy`[@numpy], `scipy`[@scipy], `matplotlib`[@matplotlib] and `pandas`[@pandas]).
```

Comment:
```suggestion
The `sunpy` package is designed to be extensible, which means that it is easy to add support for additional instruments or data sources. It relies heavily on the `astropy` [@astropy] Python package as well as the scientific python stack (e.g. `numpy` [@numpy], `scipy` [@scipy], `matplotlib` [@matplotlib] and `pandas` [@pandas]).
```

The rendered version of the paper looks a little funny without spaces between the package names and the citations (see [this version](https://github.com/openjournals/joss-papers/blob/joss.01832/joss.01832/10.21105.joss.01832.pdf)) so suggest adding those here.